addons/customize-avatar-border/addon.json

```diff
@@ -95,9 +99,44 @@
           "fill-transparent": true
         }
       }
+    },
+    {
+      "name": "Custom shape",
+      "id": "rounding",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Shape",
+      "id": "round-style",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "square",
+          "name": "Square"
+        },
+        {
+          "id": "round",
+          "name": "Rounded square"
+        },
+        {
+          "id": "circle",
+          "name": "Circle"
+        }
+      ],
+      "default": "round",
+      "if": {
+        "settings": {
+          "rounding": true
+        }
+      }
```

Comment:
Sure. Another option is to keep square/round/circle, but use a `customCssVariable`.

Reply:
I'll try that.

I could also reveal a percentage setting when the user selects "Rounded square", to allow for more precise customization. It would sort of act like an anything-in-between setting. Would that be good?

Reply:
I think just percentage setting and then maybe adding square/circle presets would be better.

Reply:
Good idea, but presets are supposed to be for the entire addon, right? Can I make a drop-down menu without adding presets?

Reply:
I haven't checked, but I don't think so. Could you make an issue for that?

Reply:
I'm getting the hang of this!

Reply:
I'm trying to only show the percentage setting when both `rounding` is true and `round-style` is "round", but it shows the setting when at least one is true. Is this intentional? How do I make this an AND instead of an OR?

Reply:
Here's my progress:

- [x] Profile page
- [ ] Messages page
- [ ] Project pages
- [ ] Studio pages
- [ ] Other pages